src/utils/Client.ts

```diff
@@ -154,6 +152,20 @@ export class Client extends Discord.Client {
 		return null;
 	}
 
+	public processAutocompleteInteraction(
+		interaction: Discord.AutocompleteInteraction
+	): boolean {
+		const command = this.slashCommands.find(
+			(c) => c.name === interaction.commandName
+		);
+
+		if (!command) return false;
+
+		command.executeAutocomplete(interaction, this);
+
+		return true;

```

Comment:
This can be easily done in one line. Not sure if eslint likes it tho.
```suggestion
		return Boolean(this.slashCommands.find(
			(c) => c.name === interaction.commandName
		)?.executeAutocomplete(interaction, this));
```

Reply:
![image](https://user-images.githubusercontent.com/71493060/149667900-4d7b0a7e-3621-46c6-909d-e8c1d96f81e0.png)
No issues so far.

Reply:
🚀